audbcards/core/dataset.py

```diff
@@ -274,6 +292,283 @@ def version_link(self) -> str:
         url = f'{github}/{self.name}/blob/{branch}/CHANGELOG.md'
         return f'`{self.version} <{url}>`__'
 
+    """Additions to class """
```

Comment:
I have added all the properties to the superclass because I couldn't use inheritance in order to store them in a derived class.  This post suggests one alternative: 

https://realpython.com/python-getter-setter/#using-inheritance-getter-and-setters-vs-properties

There might be other, nicer, or more beautiful alternatives.  Currently, the data are transported to the `Datacard` by using a `Dataset` object in the construtor.

Reply:
I think we don't need to add `properties()` to `Dataset`.
You can access the properties inside `Datacard` with:

```python
properties = [
    p for p in dir(audbcards.Dataset)
    if isinstance(getattr(audbcards.Dataset, p), property)
]
```

In addition, I would replace
```python
self.name = name
```
inside `Dataset` by a property as well:
```
@property
def name(self):
    return self.header.name
```

And I would also convert `player` to a property as there is no need to set the file manually as it should always be `self.example`.

Reply:
The properties `self.name` and `self.player` are now implemented.
My suggestion to get the properties only returns a list with the names of the properties, but we also need their current content as part of a dictionary. So, I would propose to stay with the `Dataset.properties()` method for now.

Reply:
But it would be great to remove the comment that was only meant for the review:

```suggestion
```

And afterwards sort the newly added properties alphabetically into the already existing ones.

Reply:
I believe that this has all been implemented in the #19 that is already merged?

Reply:
Yes, `self.name` and `self.player` is implemented in #19.

But my new comment also refers to the same code line and wants to remove your comment and ask to alphabetically order the properties.